### DISCUSSION TITLE
Use in a Private Repo

### DISCUSSION THREAD

**Discussion by ephraimduncan** [avitorio/outstatic#101] [Ideas]
Is there a way I can use this in a private repo? I can't create a collection when I am using a private repo.

**Comment by avitorio** [comment]
Hey,
Yes, this should work in a private repo. What error are you getting? Did you create your project with the "deploy with vercel" button?

**Reply by ephraimduncan** [reply]
I was using a private repo in an organization and it is not working

**Reply by avitorio** [reply]
Please, refer to the environment variables here: https://outstatic.com/docs/environment-variables

If you don't set the owner of the repo in the env variables, it will use your own github user id.

**Comment by city41** [comment]
I was also using a private repo. I am starting out by setting outstatic up on a branch just to try it out. It wasn't working for me because that branch did not exist remotely. Once I pushed, all was good.

**Reply by avitorio** [reply]
Thanks for sharing your experience @city41 !
